src/index.html

```diff
@@ -6,17 +6,69 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
-    />
-    <title>Moyo header</title>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>MOYO header</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
       rel="stylesheet"
-      href="./style.css"
-    />
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo-link"
+      >
+        <img
+          src="images\logo.png"
```

Comment:
The `src` attribute of the `img` tag uses backslashes (`\`). In HTML, it's more common to use forward slashes (`/`). So it should be `images/logo.png` instead of `images\logo.png`.